dashboard/app/models/applab.rb

```diff
@@ -181,4 +183,9 @@ def self.palette
     JSON
   end
 
+  def fix_examples
+    # remove nil and empty strings from examples
+    return if examples.nil?
+    self.examples = examples.select{|example| example.present?}
```

Comment:
Style/SymbolProc: Pass `&:present?` as an argument to `select` instead of a block.